test/use_case/rename_routine/RenameRoutineInteractorTest.java

```diff
@@ -55,19 +55,19 @@ public void prepareFailView(String error) {
 
 class TestDataAccess implements RenameRoutineDataAccessInterface {
 
-    int id = 1;
+    String id = "1";
     String name = "old";
 
     public TestDataAccess() {
     }
 
     @Override
-    public boolean existsById(int id) {
-        return id == this.id;
+    public boolean existsByName(String id) {
```

Comment:
i don't think this is actually testing the data access? unless I'm missing something, this overrides the dao method anyway so we don't actually run the dao method? correct me plz if i'm wrong

Reply:
also ty for deleting testestest.json

Reply:
> i don't think this is actually testing the data access? unless I'm missing something, this overrides the dao method anyway so we don't actually run the dao method? correct me plz if i'm wrong

This was testing without the DAOs to make sure it was working by itself. Also didn't have the DAOs when I was writing these.

Reply:
nice okie. will need to test those, THIS LOOKS GOOD